packages/web/src/scss/components/Grid/_tools.scss

```diff
@@ -4,7 +4,7 @@
 @use '../../tools/breakpoint';
 
 @function breakpoint-class($name, $value) {
-    @if ($value > 0) {
+    @if  $value > 0 {
```

Comment:
It comes from the [community config](https://github.com/stylelint-scss/stylelint-config-standard-scss/blob/main/index.js#L41) (which we are not afraid to tweak though). From my point of view it makes the code with at rules more consistent—no brackets anywhere:

```scss
@function breakpoint-class($name, $value) {
    @if $value > 0 {
        @return unquote($name + '--');
    }

    @return '';
}

@mixin gutters($gutters, $breakpoints) {
    @each $breakpoint-name, $breakpoint-value in $breakpoints {
        @if not map.has-key($gutters, $breakpoint-name) {
            @error 'Invalid breakpoint specified! #{$breakpoint-name} doesn\'t exist. Use one of #{map.keys($breakpoints)}';
        }
        @include breakpoint.up($breakpoint-value) {
            gap: map.get($gutters, $breakpoint-name);
        }
    }
}

@mixin equal-columns($grid-equal-columns, $breakpoints) {
    @each $breakpoint-name, $breakpoint-value in $breakpoints {
        @include breakpoint.up($breakpoint-value) {
            @each $option in $grid-equal-columns {
                .Grid--#{breakpoint-class($breakpoint-name, $breakpoint-value)}cols-#{$option} {
                    grid-template-columns: repeat(#{$option}, 1fr);
                }
            }
        }
    }
}
```

https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/scss/components/Grid/_tools.scss

Reply:
Well using ```scss
@function breakpoint-class($name, $value) {
    @if $value > 0 {
        @return unquote($name + '--');
    }

    @return '';
}
``` as an example, where the only place missing brackets is `@if` is a little bit funny to me :-)

Ok, the point was to use brackets everywhere or nowhere (like python does - it relies on tabbing). So it is weird not using single parenthesis and using only double parenthesis.

Otherwise I think I am ok with this and I will used to it. You are the authority in this case.